src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -2191,8 +2398,24 @@ mgp_error mgp_graph_create_edge(mgp_graph *graph, mgp_vertex *from, mgp_vertex *
         if (!MgpGraphIsMutable(*graph)) {
           throw ImmutableObjectException{"Cannot create an edge in an immutable graph!"};
         }
+        auto edge =
+            std::visit(memgraph::utils::Overloaded{
```

Comment:
Use one lambda with auto*. Remove Overloaded

Reply:
I think I can't use auto* since in one case I am sending VertexAccessor and in other SubgraphVertexAccessor. And also I am doing MG_ASSERT on correct type for SubgraphDbAccessor so that inside that function I have SubgraphVertexAccessor.